### DISCUSSION TITLE
[UNSTABLE] All players respawn to the coalition team in deathmatch

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#14281] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If a player respawns in deathmatch, they always respawn to the coalition team. Even if they died as part of the separatist team.

### Reproduction steps

1. Start outpost deathmatch on the separatist side
2. Die
3. Respawn
4. You dirty traitor.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and fix tested to be working correctly. Fix coming in the next Unstable build. Closing.
